src/plugin.py

```diff
@@ -202,11 +238,11 @@ def _force_utc(self):
     async def get_game_time(self, game_id: str, context: Any) -> GameTime:
         game_time = context.get(game_id)
         if game_time is None:
-            raise UnknownError("Game {} not owned".format(game_id))
+            logging.exception("Game {} not owned".format(game_id))
         return game_time
 
     async def _get_game_times_dict(self) -> Dict[str, GameTime]:
-        games = await self._client.get_games(self._steam_id)
+        games = self._own_games
```

Comment:
use cache instead of api

Reply:
This method is called be prepare game times context which gets a list of ids from galaxy

if we want to use cache we should first check if all the games in the list of ids are present in the cache, if not we should hit steam api to be sure we have the freshest results.